source/index.txt

```diff
@@ -77,6 +78,12 @@ Troubleshooting
 For solutions to issues you might encounter when using the driver,
 see the :ref:`pymongo-troubleshooting` section.
 
+Issues & Help
+-------------
+
+Learn how to report bugs, contribute to the driver, and to find help in the
```

Comment:
S: I don't think you need to repeat "to" here (since the list starts with "how to ...")
```suggestion
Learn how to report bugs, contribute to the driver, and find help in the
```